src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicPlatformDetectionTests.cs

```diff
@@ -20,20 +21,33 @@ public void UnsupportedPlatforms_ThrowsPlatformNotSupportedException()
             Assert.ThrowsAsync<PlatformNotSupportedException>(async () => await CreateQuicConnection(new IPEndPoint(IPAddress.Loopback, 0)));
         }
 
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/73290", typeof(PlatformDetection), nameof(PlatformDetection.IsSingleFile))]
```

Comment:
Based on this run: https://dev.azure.com/dnceng-public/public/_build/results?buildId=158551&view=ms.vss-test-web.build-test-results-tab&runId=3276022&resultId=132009&paneView=debug
it looks like this part works.